portfolio/src/main/java/com/google/sps/servlets/StoreCard.java

```diff
@@ -52,8 +54,11 @@ public StoreCard(PlacesSearchResult store, GeoApiContext geoApiContext) {
 
         // If the store has their hours available, check if they are open.
         OpeningHours openingHours = store.openingHours;
-        if (openingHours != null) {
+        try {
             openNow = openingHours.openNow;
-        } 
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Add a comment to say why we're only printing the stacktrace i

Reply:
Yeah, I was wondering if anything else is needed? Because otherwise the value just stays at false, so a catch isn't really necessary behavior-wise.